include/zephyr/fs/fs.h

```diff
@@ -605,6 +605,23 @@ int fs_stat(const char *path, struct fs_dirent *entry);
  */
 int fs_statvfs(const char *path, struct fs_statvfs *stat);
 
+#if defined(CONFIG_FILE_SYSTEM_MKFS)
+
+/**
+ * @brief Create fresh file system
+ *
+ * @param fs_type Type of file system to create
+ * @param dev_id Id of storage device
+ * @param cfg Backend dependent init object
```

Comment:
Note that this is optional and may be NULL, in which case the default configuration will be used.